src/v2/BaseLoginRouter.js

```diff
@@ -32,11 +32,21 @@ import AppState from './models/AppState';
 import idx from 'idx';
 
 
-const introspectStateToken = (settings) => {
+const introspect = (settings) => {
```

Comment:
I would like to maintain the separation between introspect and interact calls..

create two functions introspect and interact that returns idx.start()

create another calling function startLoginFlow() that calls the correct function from above

Reply:
interact is called inside idx.start(), so the widget doesn't explicitly call it.  If you mean two function calls to define the different params to idx.start(), then ignore this comment from me.

Reply:
yes, for the purpose of clarity for devs.